infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultClusterManager.java

```diff
@@ -49,7 +49,7 @@
 
     Thread thread = new Thread(siteManagerCallback);
 
-    public void start() {
+    public void register() {
```

Comment:
this is not doing any registration...?  what does this method do?

Reply:
It start to talk to the real ClusterManager and get the topology information.